src/main/java/uk/gov/hmcts/darts/common/entity/Media.java

```diff
@@ -62,4 +62,6 @@ public class Media {
     @Column(name = "i_version_label")
     private Short version;
 
+    @Column(name = "i_storage_location", length = 64)
```

Comment:
This column doesn't match the migration sql below where it's named `storage_location`.
We were also going to drop the length attribute.
But suspect @hemantasharma1129 / the leads were wanting a new entity close to what @garethmorriscgi did in `audio` https://github.com/hmcts/darts-api/pull/81

Reply:
We have some inconsistencies in the format of the tables we are creating. Some of them are derived from Barry's work (like here) and some we have simply made up (possibly because it doesn't exist in legacy darts). I think we need to decide which approach we are going with CC @jackmaloney @hemantasharma1129